e2e_tests/tests/cluster/test_logging.py

```diff
@@ -63,10 +62,10 @@ def test_trial_logs() -> None:
 @pytest.mark.parametrize(
     "task_type,task_config,log_regex",
     [
-        (command.TaskTypeCommand, {"entrypoint": ["echo", "hello"]}, re.compile("^.*hello.*$")),
```

Comment:
hmm why?

Reply:
because the CLI is not a library.  It should _never_ be imported from outside the `cli/` directory.

Importing it for a few static strings is a violation of the application-nature of the CLI, with no particular benefit.  I don't think these strings are going to change in the next 10 years.

Reply:
I didn't check where the `command` namespace was coming from. makes sense.